xed/xed-highlight-mode-selector.c

```diff
@@ -244,7 +244,11 @@ static void
 on_entry_realized (GtkWidget                *entry,
                    XedHighlightModeSelector *selector)
 {
+   if(gtk_widget_is_focus(selector)){
     gtk_widget_grab_focus (entry);
+   } else {
+    gtk_widget_grab_focus (selector);
+   }
```

Comment:
I didn't test this yet but the formatting doesn't match the rest of xeds style. 
`if (gtk_widget_is_focus (seleector)
{
    gtk_widget_grab_focus (entry);
}
else
{
    gtk_widget_grab_focus (selector);
}`

Reply:
(shakes fist) VIM!!!!

Fixed the formatting to match up.